plugins/action_modify_record/src/modifyRecord.test.ts

```diff
@@ -260,6 +273,169 @@ test('Test: Check creating of application join record', () => {
       values: [result.output.user.id],
     })
 
-    expect(queryResult.rows).toEqual([{ application_id: 4000, id: 6, user_id: 21 }])
+    expect(queryResult.rows).toEqual([{ application_id: 1, id: 6, user_id: 5 }])
   })
 })
+
+test('Test: add single record to new table', () => {
+  return modifyRecord({
+    parameters: { tableName: 'test', name: 'Hello', amount: 5, date: '1999-12-22' },
+    ...extraParameters,
+  }).then((result: any) => {
+    expect(result).toEqual({
+      status: ActionQueueStatus.Success,
+      error_log: '',
+      output: {
+        data_table_test: {
+          id: 1,
+          name: 'Hello',
+          amount: 5,
+          date: new Date('1999-12-21T11:00:00.000Z'),
+        },
+      },
+    })
+  })
+})
+
+test('Test: add multiple records to new table', () => {
+  return modifyRecord({
+    parameters: {
+      tableName: 'test2',
+      records: [
+        { name: 'New Record', amount: 5, isCompleted: true },
+        { name: 'Another record', amount: 10, isCompleted: false },
+        { name: 'Another record', floatingValue: 1.5 },
+      ],
+    },
+    ...extraParameters,
+  }).then((result: any) => {
+    expect(result).toEqual({
+      output: {
+        records: [
+          {
+            status: 'SUCCESS',
+            error_log: '',
+            output: {
+              data_table_test_2: {
+                id: 1,
+                name: 'New Record',
+                amount: 5,
+                is_completed: true,
+              },
+            },
+          },
+          {
+            status: 'SUCCESS',
+            error_log: '',
+            output: {
+              data_table_test_2: {
+                id: 2,
+                name: 'Another record',
+                amount: 10,
+                is_completed: false,
+              },
+            },
+          },
+          {
+            status: 'SUCCESS',
+            error_log: '',
+            output: {
+              data_table_test_2: {
+                id: 3,
+                name: 'Another record',
+                amount: null,
+                is_completed: null,
+                floating_value: 1.5,
+              },
+            },
+          },
+        ],
+      },
+      status: 'SUCCESS',
+      error_log: '',
+    })
+  })
+})
+
+test('Test: add multiple records to multiple tables, using keyMap', () => {
+  return modifyRecord({
+    parameters: {
+      tableName: 'test2',
+      records: [
+        { tableName: 'test3', fullname: 'New Record', amount: 5, isCompleted: true },
```

Comment:
Is this really needed ? Being able to edit multiple tables in one action, i can see potential use case for editing multiple records in a table, but for multiple tables ? 
If it's just ability to updateMultiple records in one table, then `updateMultipleRecords` would be a bit simpler

Reply:
Sure, there's not really an obvious use case for updating/adding to multiple tables in a single action, but is there any downside to it being possible? Like, I would have to do *more* work to not allow it.

We should probably have a test case that doesn't touch multiple tables though, so I'll add that.